pubsub.go

```diff
@@ -597,3 +631,15 @@ func WithDatastore(datastore ds.Datastore) Option {
 		return nil
 	}
 }
+
+func formatKey(key string) string {
+	ns, k, err := record.SplitKey(key)
+	if err != nil || ns != "ipns" {
+		log.Error(err)
```

Comment:
We should not error on non-ipns namespaces. 
Mind changing this so we format key only if record namespace is `ipns`, and leave it as-is for everything else?

Reply:
Fixed.